static-site/app/styles/globals.css

```diff
@@ -0,0 +1,141 @@
+/* for debugging: add a border to all other elements on the page */
+/* body * { border: 1px solid red; } */
+
+:root {
+  --titleColor0: #4377cd; /* N */
+  --titleColor1: #5097ba; /* E */
+  --titleColor2: #63ac9a; /* X */
+  --titleColor3: #7cb879; /* T */
+  --titleColor4: #9abe5c; /* S */
+  --titleColor5: #b9bc4a; /* T */
+  --titleColor6: #d4b13f; /* R */
+  --titleColor7: #e49938; /* A */
+  --titleColor8: #e67030; /* I */
+  --titleColor9: #de3c26; /* N */
+
+  --blue: #5097ba;
+  --darkGrey: #333;
+  --errorRed: #DE3C26;
+  --mediumGrey: #888;
+  --lightGrey: #ccc;
+
+  --niceFontSize: 16px;
+  --niceLineHeight: 1.6;
+
+  --tightLineHight: 1.4;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  /* We use nextjs' next/font to manage our primary font (Lato). `var(--lato)` is defined in app/layout.tsx */
+  font-family: var(--lato), "Helvetica Neue", sans-serif;
+  font-weight: 300;
+}
+
+html {
+  font-size: 62.5%;
+  margin: auto;
+  -webkit-overflow-scrolling: touch;
+}
```

Comment:
Maybe worth noting that these should be synced with the styles used by pages router?

https://github.com/nextstrain/nextstrain.org/blob/544b59e4a95f19ea71504cb9579fc0ecbda22330/static-site/src/styles/globals.css#L4-L15

Reply:
The differences between the old version and the new one are basically removing rules that were redundant.